descriptors/ModuleDescriptor-template.json

```diff
@@ -1057,10 +1057,14 @@
     },
     {
       "id": "oaipmhview",
-      "version": "1.0",
+      "version": "2.0",
       "handlers" : [
         {
           "methods" : [ "GET" ],
+          "pathPattern" : "/oai-pmh-view/updatedInstanceIds",
+          "permissionsRequired": ["inventory-storage.oai-pmh-view.updatedinstanceids.collection.get"]
+        }, {
+          "methods" : [ "POST" ],
```

Comment:
It accepts list of params in body

Reply:
Because it can carry list of instance's ids and their amout can me signoificant, that's why POST method was chosen. This will avoid facing limitation of query params in case with GET method.